.github/workflows/docker.yml

```
@@ -10,7 +10,7 @@ jobs:
   build:
     uses: atb-as/workflows/.github/workflows/cluster-docker-build-tag-push.yaml@v2
     with:
-      image: gcr.io/atb-mobility-platform/planner-web
+      image: europe-west1-docker.pkg.dev/amp-artifacts/amp/planner-web
```

@mjenssen Aha, so we only set the image name then?

```suggestion
      image: planner-web
```

---

No, it should be like it was without this PR

---

Then why are the builds failing? 🤔 https://github.com/AtB-AS/planner-web/actions/runs/14081026165/job/39433812339

---

This is because the common one was _also_ pushing to GCR (as mentioned in Slack earlier). This has been fixed in the common workflow now. The next action run should work